package.json

```diff
@@ -35,20 +36,40 @@
   },
   "homepage": "https://github.com/nasa/cumulus-message-adapter-js#readme",
   "devDependencies": {
+    "@ava/typescript": "^1.1.1",
```

Comment:
Unless our tests are also migrated to TS, I don't think we need `@ava/typescript`.

Reply:
ba71706